### ISSUE TITLE
[v4 iOS] Using ion-inputs "clearInput" button loses input focus

### ISSUE THREAD

**Issue by wilk-polarny** [ionic-team/ionic#17408]
# Bug Report

**Ionic version:**
[x] **4.x**

**Current behavior:**
An ion-input element with its clear button enabled (`clearInput `) will lose its input focus once the clear button is being pressed - which results in the keyboard hiding again

**Expected behavior:**
The element should not lose its focus, allowing further inputs

**Steps to reproduce:**
1. Download any starter as a base
2. Add a simple ion-input element to a list or any other structure.
3. Build and run on/for iOS (automatically uses wkwebview plugin 3.x)

![ezgif-2-b1aac907b15c](https://user-images.githubusercontent.com/819238/52377678-d9768f00-2a65-11e9-87a4-30dd6eb9d0e1.gif)

```
<ion-input clearInput value="clear me"></ion-input>
```

**Other information:**
This issue has always been present in Ionic v3, and first disappeared in Ionic 4-BETA (not sure which one). Ionic 4 stable now seems to reintroduce this problem.

**Ionic info:** 

```
Ionic:

   ionic (Ionic CLI)             : 4.10.2 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.0.1
   @angular-devkit/build-angular : 0.12.4
   @angular-devkit/schematics    : 7.2.4
   @angular/cli                  : 7.2.4
   @ionic/angular-toolkit        : 1.3.0

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 3.1.2, (and 4 other plugins)

System:

   Android SDK Tools : 25.2.2 (/Users/xxx/Library/Android/sdk)
   NodeJS            : v8.12.0 (/usr/local/bin/node)
   npm               : 6.4.1
   OS                : macOS Mojave
   Xcode             : Xcode 10.1 Build version 10B61
```

**Comment by wilk-polarny** [comment]
Update: This issue is still reproducable in 4.3.0

**Comment by wilk-polarny** [comment]
This issue has been resolved with v 4.3.0
29bb4fc
